### DISCUSSION TITLE
Database Transaction for MongoDB

### DISCUSSION THREAD

**Discussion by amit-isw** [loopbackio/loopback-next#9240] [Q&A] [answered]
Does loopback support database transaction for MongoDB?

**Comment by nflaig** [comment, marked as answer]
Yes, you can just follow the documentation [here](https://loopback.io/doc/en/lb4/Using-database-transactions.html#start-transaction). Note, that this only works if you run mongodb as a replica set and not with a standalone instance.

**Reply by amit-isw** [reply]
Thank you.
